scrapyrt/core.py

```diff
@@ -164,7 +164,7 @@ def _get_log_file_path(self):
         log_dir = os.path.join(self.log_dir, self.spider_name)
         if not os.path.exists(log_dir):
             os.makedirs(log_dir)
-        filename = datetime.datetime.now().isoformat() + '.log'
+        filename = datetime.datetime.now().strftime('%Y-%m-%dT%H;%M;%S.%f') + '.log'
```

Comment:
2nd version will produce log files with the same name for all crawls that happened at the same minute. current microsecond precision prevents this from happening.

my 2¢ - we can use dots to keep it human readable `2016-12-08T11.31.06.372041.log`. it shouldn't be an iso necessarily, it's a filename and should be human-readable after all.